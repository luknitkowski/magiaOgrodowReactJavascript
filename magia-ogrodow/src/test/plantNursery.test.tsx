import renderer from 'react-test-renderer'

import PlantNursery from '../components/plantNursery';

test('PlantNursery renders correctly', () => {
  const plantNursery = renderer.create(<PlantNursery />).toJSON();
  expect(plantNursery).toMatchSnapshot();
});

