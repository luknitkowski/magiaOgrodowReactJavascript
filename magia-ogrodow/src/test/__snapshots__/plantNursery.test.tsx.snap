// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlantNursery renders correctly 1`] = `
<div
  className="sc-dlfnbm bAyFQJ"
>
  <div
    className="sc-bdfBwQ cDzsej"
  >
    <h1
      className="sc-gsTCUz iyjhuT"
    >
      
    </h1>
  </div>
  <div
    className="sc-hKgILt fkqgQN"
  >
    <div
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      <div
        className="sc-eCssSg KqBf"
      >
        <h2>
          NASZA SZKÓŁKA
        </h2>
        <p>
          W naszej szkółce znajdziecie Państwo rośliny w pojemnikach, które można wysadzać przez cały okres wegetacyjny tj. od wczesnej wiosny do próżnej jesieni. Dokładamy wszelkich starań, aby nasz materiał roślinny był najwyższej jakości . Z każdym sezonem zwiększamy nasz asortyment o ciekawe gatunki i odmiany, których stale przybywa na rynku ogrodniczym. Zadowolenie i radość klientów to nasz priorytet.
        </p>
      </div>
    </div>
  </div>
  <div
    className="sc-jSgupP hgnalC"
  >
    <div
      className="sc-gKsewC gQrGgm"
    >
      <h3
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        Posiadamy:
      </h3>
      <ul
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div>
          <li>
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 22c4.97 0 9-4.03 9-9-4.97 0-9 4.03-9 9zM5.6 10.25c0 1.38 1.12 2.5 2.5 2.5.53 0 1.01-.16 1.42-.44l-.02.19c0 1.38 1.12 2.5 2.5 2.5s2.5-1.12 2.5-2.5l-.02-.19c.4.28.89.44 1.42.44 1.38 0 2.5-1.12 2.5-2.5 0-1-.59-1.85-1.43-2.25.84-.4 1.43-1.25 1.43-2.25 0-1.38-1.12-2.5-2.5-2.5-.53 0-1.01.16-1.42.44l.02-.19C14.5 2.12 13.38 1 12 1S9.5 2.12 9.5 3.5l.02.19c-.4-.28-.89-.44-1.42-.44-1.38 0-2.5 1.12-2.5 2.5 0 1 .59 1.85 1.43 2.25-.84.4-1.43 1.25-1.43 2.25zM12 5.5c1.38 0 2.5 1.12 2.5 2.5s-1.12 2.5-2.5 2.5S9.5 9.38 9.5 8s1.12-2.5 2.5-2.5zM3 13c0 4.97 4.03 9 9 9 0-4.97-4.03-9-9-9z"
              />
            </svg>
            <b>
              Drzewa i krzewy iglaste
            </b>
          </li>
          <p>
            Większość roślin iglastych to gatunki zimozielone, ale można znaleźć gatunki gubiące igły na zimę. Różnorodność kształtów i kolorów jest ogromna. Iglaki są nieocenionym elementem ożywiającym krajobraz zimowy.
          </p>
        </div>
        <div>
          <li>
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 22c4.97 0 9-4.03 9-9-4.97 0-9 4.03-9 9zM5.6 10.25c0 1.38 1.12 2.5 2.5 2.5.53 0 1.01-.16 1.42-.44l-.02.19c0 1.38 1.12 2.5 2.5 2.5s2.5-1.12 2.5-2.5l-.02-.19c.4.28.89.44 1.42.44 1.38 0 2.5-1.12 2.5-2.5 0-1-.59-1.85-1.43-2.25.84-.4 1.43-1.25 1.43-2.25 0-1.38-1.12-2.5-2.5-2.5-.53 0-1.01.16-1.42.44l.02-.19C14.5 2.12 13.38 1 12 1S9.5 2.12 9.5 3.5l.02.19c-.4-.28-.89-.44-1.42-.44-1.38 0-2.5 1.12-2.5 2.5 0 1 .59 1.85 1.43 2.25-.84.4-1.43 1.25-1.43 2.25zM12 5.5c1.38 0 2.5 1.12 2.5 2.5s-1.12 2.5-2.5 2.5S9.5 9.38 9.5 8s1.12-2.5 2.5-2.5zM3 13c0 4.97 4.03 9 9 9 0-4.97-4.03-9-9-9z"
              />
            </svg>
            <b>
              Drzewa i krzewy liściaste
            </b>
          </li>
          <p>
            Od zawsze zajmowały szczególne miejsce w otoczeniu ludzi. Tworzą urozmaiconą grupę. Są to rośliny o różnych formach pokrojowych, o barwnych i wymyślnych liściach, dekoracyjnych kwiatach i owocach. Zastosowanie ich to wprowadzenie życia do ogrodu, pozbycie się nudy i monotonii.
          </p>
        </div>
        <div>
          <li>
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 22c4.97 0 9-4.03 9-9-4.97 0-9 4.03-9 9zM5.6 10.25c0 1.38 1.12 2.5 2.5 2.5.53 0 1.01-.16 1.42-.44l-.02.19c0 1.38 1.12 2.5 2.5 2.5s2.5-1.12 2.5-2.5l-.02-.19c.4.28.89.44 1.42.44 1.38 0 2.5-1.12 2.5-2.5 0-1-.59-1.85-1.43-2.25.84-.4 1.43-1.25 1.43-2.25 0-1.38-1.12-2.5-2.5-2.5-.53 0-1.01.16-1.42.44l.02-.19C14.5 2.12 13.38 1 12 1S9.5 2.12 9.5 3.5l.02.19c-.4-.28-.89-.44-1.42-.44-1.38 0-2.5 1.12-2.5 2.5 0 1 .59 1.85 1.43 2.25-.84.4-1.43 1.25-1.43 2.25zM12 5.5c1.38 0 2.5 1.12 2.5 2.5s-1.12 2.5-2.5 2.5S9.5 9.38 9.5 8s1.12-2.5 2.5-2.5zM3 13c0 4.97 4.03 9 9 9 0-4.97-4.03-9-9-9z"
              />
            </svg>
            <b>
              Drzewa i krzewy owocowe
            </b>
          </li>
          <p>
            Dostarczają smacznych owoców, które można wykorzystać na różne sposoby. Zbiory z własnych drzewek i krzewów smakują lepiej i sprawiają dużo radości. Większość drzewek zaszczepione są na podkładkach karłowych, które można wysadzać nawet w najmniejszych ogrodach.
          </p>
        </div>
        <div>
          <li>
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 22c4.97 0 9-4.03 9-9-4.97 0-9 4.03-9 9zM5.6 10.25c0 1.38 1.12 2.5 2.5 2.5.53 0 1.01-.16 1.42-.44l-.02.19c0 1.38 1.12 2.5 2.5 2.5s2.5-1.12 2.5-2.5l-.02-.19c.4.28.89.44 1.42.44 1.38 0 2.5-1.12 2.5-2.5 0-1-.59-1.85-1.43-2.25.84-.4 1.43-1.25 1.43-2.25 0-1.38-1.12-2.5-2.5-2.5-.53 0-1.01.16-1.42.44l.02-.19C14.5 2.12 13.38 1 12 1S9.5 2.12 9.5 3.5l.02.19c-.4-.28-.89-.44-1.42-.44-1.38 0-2.5 1.12-2.5 2.5 0 1 .59 1.85 1.43 2.25-.84.4-1.43 1.25-1.43 2.25zM12 5.5c1.38 0 2.5 1.12 2.5 2.5s-1.12 2.5-2.5 2.5S9.5 9.38 9.5 8s1.12-2.5 2.5-2.5zM3 13c0 4.97 4.03 9 9 9 0-4.97-4.03-9-9-9z"
              />
            </svg>
            <b>
              Pnącza
            </b>
          </li>
          <p>
            To rośliny szczególne wytwarzające dużą masę zieleni lub kwiatów. Potrafią zdynamizować każdą przestrzeń, zapełniają trzeci wymiar tworząc kolorowe, pionowe ściany
          </p>
        </div>
        <div>
          <li>
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 22c4.97 0 9-4.03 9-9-4.97 0-9 4.03-9 9zM5.6 10.25c0 1.38 1.12 2.5 2.5 2.5.53 0 1.01-.16 1.42-.44l-.02.19c0 1.38 1.12 2.5 2.5 2.5s2.5-1.12 2.5-2.5l-.02-.19c.4.28.89.44 1.42.44 1.38 0 2.5-1.12 2.5-2.5 0-1-.59-1.85-1.43-2.25.84-.4 1.43-1.25 1.43-2.25 0-1.38-1.12-2.5-2.5-2.5-.53 0-1.01.16-1.42.44l.02-.19C14.5 2.12 13.38 1 12 1S9.5 2.12 9.5 3.5l.02.19c-.4-.28-.89-.44-1.42-.44-1.38 0-2.5 1.12-2.5 2.5 0 1 .59 1.85 1.43 2.25-.84.4-1.43 1.25-1.43 2.25zM12 5.5c1.38 0 2.5 1.12 2.5 2.5s-1.12 2.5-2.5 2.5S9.5 9.38 9.5 8s1.12-2.5 2.5-2.5zM3 13c0 4.97 4.03 9 9 9 0-4.97-4.03-9-9-9z"
              />
            </svg>
            <b>
              Byliny
            </b>
          </li>
          <p>
            Są zielonymi roślinami wieloletnimi zimującymi w gruncie i wyrastającymi ponownie każdego roku. Różnorodność bylin pozwala zagospodarować każdy zakątek w ogrodzie. Można dobrać byliny dobrze czujące się w słońcu, cieniu, na glebach suchych i podmokłych.
          </p>
        </div>
        <div>
          <li>
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 22c4.97 0 9-4.03 9-9-4.97 0-9 4.03-9 9zM5.6 10.25c0 1.38 1.12 2.5 2.5 2.5.53 0 1.01-.16 1.42-.44l-.02.19c0 1.38 1.12 2.5 2.5 2.5s2.5-1.12 2.5-2.5l-.02-.19c.4.28.89.44 1.42.44 1.38 0 2.5-1.12 2.5-2.5 0-1-.59-1.85-1.43-2.25.84-.4 1.43-1.25 1.43-2.25 0-1.38-1.12-2.5-2.5-2.5-.53 0-1.01.16-1.42.44l.02-.19C14.5 2.12 13.38 1 12 1S9.5 2.12 9.5 3.5l.02.19c-.4-.28-.89-.44-1.42-.44-1.38 0-2.5 1.12-2.5 2.5 0 1 .59 1.85 1.43 2.25-.84.4-1.43 1.25-1.43 2.25zM12 5.5c1.38 0 2.5 1.12 2.5 2.5s-1.12 2.5-2.5 2.5S9.5 9.38 9.5 8s1.12-2.5 2.5-2.5zM3 13c0 4.97 4.03 9 9 9 0-4.97-4.03-9-9-9z"
              />
            </svg>
            <b>
              Rośliny jednoroczne
            </b>
          </li>
          <p>
            Grupa roślin ozdobnych. Ich rozwój przebiega przez jeden sezon wegetacyjny, co oznacza, że potrzebują one więcej uwagi i naszego zaangażowania, dzięki temu pokażą nam swoje najpiękniejsze oblicze.
          </p>
        </div>
      </ul>
    </div>
  </div>
  <div
    className="sc-bdfBwQ gRJaMH"
  >
    <h1
      className="sc-gsTCUz iyjhuT"
    >
      
    </h1>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
    >
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="MuiGrid-root makeStyles-gridWithImage-4 MuiGrid-item"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <img
              alt="plant"
              height="100%"
              src="/images/plant/2-min-min.jpg"
              width="100%"
            />
          </div>
        </div>
      </div>
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="MuiGrid-root makeStyles-gridWithImage-4 MuiGrid-item"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <img
              alt="plant"
              height="100%"
              src="/images/plant/3-min-min.jpg"
              width="100%"
            />
          </div>
        </div>
      </div>
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="MuiGrid-root makeStyles-gridWithImage-4 MuiGrid-item"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <img
              alt="plant"
              height="100%"
              src="/images/plant/4-min-min.jpg"
              width="100%"
            />
          </div>
        </div>
      </div>
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="MuiGrid-root makeStyles-gridWithImage-4 MuiGrid-item"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <img
              alt="plant"
              height="100%"
              src="/images/plant/5-min-min.jpg"
              width="100%"
            />
          </div>
        </div>
      </div>
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="MuiGrid-root makeStyles-gridWithImage-4 MuiGrid-item"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <img
              alt="plant"
              height="100%"
              src="/images/plant/6-min-min.jpg"
              width="100%"
            />
          </div>
        </div>
      </div>
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="MuiGrid-root makeStyles-gridWithImage-4 MuiGrid-item"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <img
              alt="plant"
              height="100%"
              src="/images/plant/7-min-min.jpg"
              width="100%"
            />
          </div>
        </div>
      </div>
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="MuiGrid-root makeStyles-gridWithImage-4 MuiGrid-item"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <img
              alt="plant"
              height="100%"
              src="/images/plant/8-min-min.jpg"
              width="100%"
            />
          </div>
        </div>
      </div>
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="MuiGrid-root makeStyles-gridWithImage-4 MuiGrid-item"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <img
              alt="plant"
              height="100%"
              src="/images/plant/9-min-min.jpg"
              width="100%"
            />
          </div>
        </div>
      </div>
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="MuiGrid-root makeStyles-gridWithImage-4 MuiGrid-item"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <img
              alt="plant"
              height="100%"
              src="/images/plant/11-min-min.jpg"
              width="100%"
            />
          </div>
        </div>
      </div>
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="MuiGrid-root makeStyles-gridWithImage-4 MuiGrid-item"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <img
              alt="plant"
              height="100%"
              src="/images/plant/13-min-min.jpg"
              width="100%"
            />
          </div>
        </div>
      </div>
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="MuiGrid-root makeStyles-gridWithImage-4 MuiGrid-item"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <img
              alt="plant"
              height="100%"
              src="/images/plant/14-min-min.jpg"
              width="100%"
            />
          </div>
        </div>
      </div>
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="MuiGrid-root makeStyles-gridWithImage-4 MuiGrid-item"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <img
              alt="plant"
              height="100%"
              src="/images/plant/15-min-min.jpg"
              width="100%"
            />
          </div>
        </div>
      </div>
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="MuiGrid-root makeStyles-gridWithImage-4 MuiGrid-item"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <img
              alt="plant"
              height="100%"
              src="/images/plant/16-min-min.jpg"
              width="100%"
            />
          </div>
        </div>
      </div>
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="MuiGrid-root makeStyles-gridWithImage-4 MuiGrid-item"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <img
              alt="plant"
              height="100%"
              src="/images/plant/17-min-min.jpg"
              width="100%"
            />
          </div>
        </div>
      </div>
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="MuiGrid-root makeStyles-gridWithImage-4 MuiGrid-item"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <img
              alt="plant"
              height="100%"
              src="/images/plant/18-min-min.jpg"
              width="100%"
            />
          </div>
        </div>
      </div>
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="MuiGrid-root makeStyles-gridWithImage-4 MuiGrid-item"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <img
              alt="plant"
              height="100%"
              src="/images/plant/19-min-min.jpg"
              width="100%"
            />
          </div>
        </div>
      </div>
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="MuiGrid-root makeStyles-gridWithImage-4 MuiGrid-item"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <img
              alt="plant"
              height="100%"
              src="/images/plant/20-min-min.jpg"
              width="100%"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="sc-iBPRYJ ctNbMr"
  >
    <p
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      Jeśli chcesz zobaczyć wiecej zdjęć to zapraszamy na naszą stronę na facebooku! Link znajdziesz na dole strony.
    </p>
    <div
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      <span
        aria-disabled={false}
        aria-label="upload picture"
        className="MuiButtonBase-root MuiIconButton-root sc-fubCfw upDeo MuiIconButton-colorPrimary"
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="MuiIconButton-label"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root sc-pFZIQ btXyVc"
            focusable="false"
            style={
              Object {
                "height": "100px",
                "width": "100px",
              }
            }
            viewBox="0 0 24 24"
          >
            <path
              d="M7.41 8.59L12 13.17l4.59-4.58L18 10l-6 6-6-6 1.41-1.41z"
            />
          </svg>
        </span>
      </span>
    </div>
  </div>
  <div
    className="sc-bdfBwQ ilUtcr"
  >
    <h1
      className="sc-gsTCUz iyjhuT"
    >
      
    </h1>
  </div>
</div>
`;
